CMP1903_A1_2324/Game.cs

```diff
@@ -10,10 +12,55 @@ internal class Game
     {
         /*
          * The Game class should create three die objects, roll them, sum and report the total of the three dice rolls.
+         
+         *
+         *
          *
          * EXTRA: For extra requirements (these aren't required though), the dice rolls could be managed so that the
          * rolls could be continous, and the totals and other statistics could be summarised for example.
          */
+        //
+       
+        public Die Die1;
+        public Die Die2;
+        public Die Die3;
+        public int sum_of_dice;
```

Comment:
Public integers should be written in camel case: sumOfDice

Reply:
(It should be SumOfDice, public variables should be in Pascal Case)